src/Estimators/MagnetizationDensityInput.h

```diff
@@ -0,0 +1,129 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2023 QMCPACK developers.
+//
+// File developed by: Raymond Clay, rclay@sandia.gov, Sandia National Laboratories
+//
+// File created by: Raymond Clay, rclay@sandia.gov, Sandia National Laboratories
+//////////////////////////////////////////////////////////////////////////////////////
+
+
+#ifndef QMCPLUSPLUS_MAGNETIZATION_DENSITY_INPUT_H
+#define QMCPLUSPLUS_MAGNETIZATION_DENSITY_INPUT_H
+
+#include "Configuration.h"
+#include "OhmmsData/ParameterSet.h"
+#include "Containers/OhmmsPETE/TinyVector.h"
+#include "Estimators/InputSection.h"
+namespace qmcplusplus
+{
+class MagnetizationDensity;
+
+class MagnetizationDensityInput
+{
+public:
+  enum class Integrator
+  {
+    SIMPSONS,
+    MONTECARLO
+  };
+
+  inline static const std::unordered_map<std::string, std::any> lookup_input_enum_value{{"integrator-simpsons",
```

Comment:
clang-format?  or is responsible for this?

Reply:
clang-format is responsible for that.  Improper application of clang-format not ruled out.

Reply:
Double checked the clang-format output.  Running from src, that's the output that should be consistent with .clang-format.  I can manually format it if you like.